src/player_team.ts

```diff
@@ -990,7 +990,8 @@ class Team {
           let curAtk = ping.source.getAtk(pm, awoke);
           curAtk = Round.UP(curAtk * baseMultiplier);
           if (ping.isSub) {
```

Comment:
Change the conditional so that it doesn't do any multiplication if the main attribute is NONE:  Remove the change from below since "divisor" isn't counted on a subattribute-only monster.

if (ping.isSub && ping.source.getAttribute() != Attribute.JAMMER) {

Reply:
Change made.